sdks/java/core/src/main/java/org/apache/beam/sdk/values/PValueBase.java

```diff
@@ -22,11 +22,14 @@
 import java.util.Collections;
 import java.util.Map;
 import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.annotations.Internal;
 import org.apache.beam.sdk.transforms.PTransform;
 import org.apache.beam.sdk.util.NameUtils;
 
 /**
- * A {@link PValueBase} is an abstract base class that provides
+ * <b><i>For internal use. No backwards compatibility guarantees.</i></b>
+ *
+ * <p>A {@link PValueBase} is an abstract base class that provides
```

Comment:
FWIW I don't think we need "sensible defaults" for an non-user-facing interface with one implementation. But no need to mess with that now, obvs.